zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -1423,21 +1423,99 @@ paths:
         "500":
           $ref: "#/components/responses/InternalServerError"
 
+  /variables:
+    post:
+      tags:
+        - Variable
+      operationId: createVariable
+      summary: Create variable
+      description: |
+        Create cluster level variables
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/CreateVariableRequest"
+      responses:
+        "201":
+          description: >
+            The variable creation status.
+        "400":
+          $ref: "#/components/responses/InvalidData"
+        "401":
+          $ref: "#/components/responses/Unauthorized"
+        "403":
+          $ref: "#/components/responses/Forbidden"
+        "500":
+          $ref: "#/components/responses/InternalServerError"
+
   /variables/{variableKey}:
-    get:
+    parameters:
+      - name: variableKey
+        in: path
+        required: true
+        description: The variable key.
+        schema:
+          type: string
+    patch:
       tags:
         - Variable
-      operationId: getVariable
-      summary: Get variable
+      operationId: updateVariable
+      summary: Update variables
```

🔧 We're just updating one variable here.

```suggestion
      summary: Update variable
```